### DISCUSSION TITLE
種族の調整案

### DISCUSSION THREAD

**Discussion by Aquashrimp** [hengband/hengband#323] [Ideas]
各種族の調整案を出してみるスレッドです

これは種族に限った話ではありませんが、レイシャルパワーの成功率計算に使う値はステータスの素の値ではなく、魔法などと同じように装備のpvalも合算して計算するのはどうでしょうか？

**Comment by HiddenNameZ** [comment]
いくつか出します
- 骸骨に地獄耐性を付与する
- 吸血鬼を光源もちではなく、忍者と同じように暗闇を見通せるようにする
- フレーバー上の観点からアンドロイドは時間逆転属性に耐性があってもいい気がします 他にもバルログとかアルコンあたりも持ってそうですけどまあ乱発してもしたかがないので
- これは前に出ていた気もしますが、(ハーフ)オークの恐怖除去が使い物にならないのでヒーロー化にするか恐怖耐性に変更する
- ソースを追加する必要がありそうですが、サイクロプスの轟音耐性がわりと意味不明なので、それを取り除いて盲目関連で特殊な処理が欲しい(たとえば、盲目耐性がない場合どんな近接攻撃でも一定確率で短時間盲目になるとか)

**Comment by iks3** [comment]
### ハーフトロルについて

|腕力 | 知能 | 賢さ | 器用 | 耐久 | 魅力 | 解除 | 魔道 | 魔防 | 隠密 | 探索 | 知覚 | 打撃 | 射撃 | HD | 赤外 | 経験 | 援軍|
|-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --|
| 4 | -4 | -1 | -3 | 3 | -3 | -5 | -8 | -5 | -2 | -1 | 5 | 20 | -10 | 12 | 3 | 25 | T|


|酸 | 電 | 火 | 冷 | 毒 | 閃 | 暗 | 破 | 盲 | 乱 | 轟 | 獄 | 因 | 沌 | 劣 | 恐 | 麻 | 透 | 経 | テ | 遅 | 急 | 浮|
|-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --|
| - | - | - | - | - | - | - | - | - | - | - | - | - | - | - | - | - | - | - | - | - | 15 | -|

戦士の場合に限り, レベル15で遅消化を得る. 狂戦士化(レベル10,コスト12,賢さ).

**Reply by iks3** [reply]
・レベル1から急回復があってもいいのではないか？
・レベル1から狂戦士化が使えてもいいのではないか？
・レイシャルか何かで*急速回復*感が出せないか？
・強靭な爪のおかげでマーシャルアーツ時殺戮修正追加みたいなの？
・重武器の使用にボーナス？

**Comment by iks3** [comment]
### ハーフオークについて

|腕力 | 知能 | 賢さ | 器用 | 耐久 | 魅力 | 解除 | 魔道 | 魔防 | 隠密 | 探索 | 知覚 | 打撃 | 射撃 | HD | 赤外 | 経験 | 援軍|
|-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --|
|2 | -1 | 0 | 0 | 1 | -2 | -3 | -3 | -2 | -1 | 0 | 7 | 12 | -5 | 10 | 3 | 10 | o|






|酸 | 電 | 火 | 冷 | 毒 | 閃 | 暗 | 破 | 盲 | 乱 | 轟 | 獄 | 因 | 沌 | 劣 | 恐 | 麻 | 透 | 経 | テ | 遅 | 急 | 浮|
|--|--|--|--|--|--|--|--|--|--|--|--|--|--|--|--|--|--|--|--|--|--|--|
|- | - | - | - | - | - | 1 | - | - | - | - | - | - | - | - | - | - | - | - | - | - | - | -|

恐怖除去(レベル3,コスト5,賢さ).

**Reply by iks3** [reply]
・恐怖除去が使い物にならないのでヒーロー化にするか恐怖耐性に変更する？

**Reply by ghost** [reply]
オーク=愚鈍みたいなイメージでしたがwikiでおさらいしてみると
>手先が器用で、歯車や機械に興味を持ち、やっとこやつるはしや斧などの他「人をたくさん殺す機械」以外は何も生み出さず

<https://ja.wikipedia.org/wiki/%E3%82%AA%E3%83%BC%E3%82%AF_(%E3%83%88%E3%83%BC%E3%83%AB%E3%82%AD%E3%83%B3)>
という一節もあり、その辺の形質を受け継いでいるとして器用や解除の数値を上げてもいいかもと思えました。

**Reply by deskull-m** [reply]
恐怖耐性と器用、解除上昇に1票。

**Comment by HiddenNameZ** [comment]
いくつか提案
・ハイエルフ…トールキンに則るなら、目が光源になっているはずなので、吸血鬼のように永久光源+1をつけてもよいかと。さらにライトエリアのレイシャルパワーを付与してもいいのでは(ハイエルフをプレイしてる初心者が多めでライトエリアは序盤嬉しい)
・アルコン…レイシャルパワーに解呪/x解呪x(レベル20以上で使用可能、レベル40以上でx解呪xになる、コストは30くらい？、xはアスタリスクの代わり)を追加する。自前で解呪を持たない職で呪い増殖装備が使いやすくなる。
どちらも経験値補正のわりに合わないと言われることがあるのでこれくらいは強くしてもいいんじゃないでしょうか

**Reply by deskull-m** [reply]
個人的にはどちらも良い案だと思います。

**Comment by deskull-m** [comment]
種族調整についてはそういえばOSDNに未消化のチケットが残っていました。
https://osdn.net/projects/hengband/ticket/37333
これもアイデアに盛り込むべきかもしれませんね。

**Comment by HiddenNameZ** [comment]
### アルコンに閃光耐性を付与する
モンスターのアルコンには閃光耐性があるので、種族にも付与してもいいと思います。閃光耐性は比較的重要ではなく、経験値修整も重いので。（ただ下位の天使モンスターには閃光耐性は有りません）

**Reply by deskull-m** [reply]
コイツも欲しい所ではありますか。ちょっと春先からの意見含めて馬鹿馬鹿で個人的備忘だけしておきます。

**Comment by ghost** [comment]
### 死体を食べるシステムの拡充
調整というより初心者・うっかりミス救済の観点で。
不注意で食糧なしでダンジョンに潜って死にかけて思いついたネタですが、バルログがh、pシンボルの死体を食料にするように他の種族でもモンスターの死体で空腹を満たすように出来ないでしょうか？
具体的には@が人間など「食料で空腹を満たす種族」の場合にB(鳥)、Ｃ(犬)、q(四足獣)などで「食用可能(と思われる)モンスターの死体」で空腹を満たせられたらなあと考えております。(食糧が尽きた@が獣肉を食べて飢えをしのぐイメージ)
ただシンボルだけで判定できない(毒を持つモンスターや実体のないモンスターなど食べるのが不自然な物もいる）とも思えますので煩雑かもしれません。

なおpやhの死体を食べるというのは(物理的には可能でも)避けてほしいです。
なんらからのペナルティ付きでとも思ったのですが、現在の変愚では徳や社会的地位が低下してもあまりプレーに変化がでないように思えましたので。
以上ご検討いただけましたら幸いです。

**Reply by deskull-m** [reply]
すぐにどうこうは言えませんが、検討には入っています。

死体を食うことやそれに関する効果は、実の所 EAT_* 系フラグがr_info向けに今も付加されてます。いつからかは未調査なのですが明らかに「食べたら作用を起こす」実装を作りかけていたことがあったようです。
それを踏まえて自分も現在の馬鹿馬鹿や昔作りかけのバリアントでこれらの効果を実際のある程度実装したことがあります。
馬鹿馬鹿では社会的道義の問題とかも併せて実装進めていますが、変愚でどこまで拡張するかは微妙なところですね。

**Reply by ghost** [reply]
>明らかに「食べたら作用を起こす」実装を作りかけていたことがあったようです。

なるほど、たしかに単純な食糧以外にもHP/MP回復、一時耐性を得るetc幅広く出来そうではありますね。
鍛冶師のように死体からエッセンスを集めて何かする職業とか妄想が広がります。
いずれ何らかの形でプレーできることを楽しみにしておきます。

**Comment by HiddenNameZ** [comment]
### ゾンビが普通にご飯を食べるようにする
ゾンビとゴーレムって色々性能似てますし、差別化やゾンビのフレーバー的にも、ゾンビには普通に食料を食べさせてもいいのではないでしょうか？
ゾンビっぽさとして、食料でも（種族問わず）死体でもおｋって感じで
あと遅消化の代わりに急速回復あるいは消化速度だけ上昇してるとそれっぽい気がします（ただ、どっちかというとバイオハザード的なゾンビになってしまいますが…）

**Reply by deskull-m** [reply]
仰る通り、よくよく考えてみると一番悪食であるべき存在な気がしますよね。他種族の食糧事情と合わせて念頭に入れたいところです。
